scene/gui/text_edit.cpp

```diff
@@ -6290,9 +6283,6 @@ void TextEdit::_bind_methods() {
 	BIND_ENUM_CONSTANT(LINE_WRAPPING_NONE);
 	BIND_ENUM_CONSTANT(LINE_WRAPPING_BOUNDARY);
 
-	// Internal.
-	ClassDB::bind_method(D_METHOD("_update_wrap_at_column", "force"), &TextEdit::_update_wrap_at_column, DEFVAL(false));
```

Comment:
This seems used in the converter, unsure if we should respect it as it's internal (even marked as such)

Reply:
This method is technically not exposed, so it shouldn't even be in the converter.

Reply:
Strange, see:
```
ERROR: Missing GDScript function in pair (_update_wrap_at - ===> _update_wrap_at_column <===)
```

Reply:
https://github.com/godotengine/godot/blob/2d0ee20ff30461b6b10f6fdfba87511a0ebc6642/editor/renames_map_3_to_4.cpp#L217

Reply:
I mean it was a wrong rename. I removed it.